_fragments/covidr-intro.Rmd

```diff
@@ -13,10 +13,10 @@ Contributions can be submitted **until May 22nd**, and the online e-Rum2020 pre-
 
 **What you will get** by participating to CovidR
 
-- All contribution will be featured in the [CovidR Gallery](https://milano-r.github.io/erum2020-covidr-contest) website, allowing the community to browse through and provide feedback.
+- All contributions will be featured in the [CovidR Gallery](https://milano-r.github.io/erum2020-covidr-contest) website, allowing the community to browse through, vote for your contribution, and provide feedback.
 - Each participant will get a CovidR badge to include in their repository.
 - We will share contributions on our social media.
-- Selected contributions will be presented at the eRum2020 pre-conference event on May 29th.
+- Selected contributions will be invited to present at the e-Rum2020 pre-conference event on May 29th.
```

Comment:
Contributions cannot actually present :). So either we invite contributors to present or contributions will be presented.

Reply:
Fixed in commit 80d6152